packages/lit-dev-content/samples/examples/motion-carousel/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<head>
+  <script type="module" src="./my-element.js"></script>
```

Comment:
Could be nice to give each example a custom filename and element name e.g. `<motion-carousel>`. Would help distinguish them when checking out the code, since they're all `<my-element>` in `my-element.js` right now.

Reply:
Done.